Xamarin.Forms.Platform.iOS/Renderers/DatePickerRenderer.cs

```diff
@@ -159,8 +161,28 @@ void UpdateDateFromModel(bool animate)
 		{
 			if (_picker.Date.ToDateTime().Date != Element.Date.Date)
 				_picker.SetDate(Element.Date.ToNSDate(), animate);
-
-			Control.Text = Element.Date.ToString(Element.Format);
+			if (Element.Format.Equals("") || Element.Format.Equals("d"))
+			{
+				NSDateFormatter dateFormatter = new NSDateFormatter();
```

Comment:
For the format strings "d" and "D", doesn't `Control.Text = Element.Date.ToString(Element.Format);` produce the same results as using the NSDateFormatter?

Reply:
No, because when region and language are set differently the Element.Format display doesn't show the proper string. So, the only way I found to get the iOS default string for a given region and language is using an NSDateFormatter.